distsql/select_result.go

```diff
@@ -68,6 +68,10 @@ type selectResult struct {
 	feedback     *statistics.QueryFeedback
 	partialCount int64 // number of partial results.
 	sqlType      string
+
+	// copPlanIDs contains all copTasks' planIDs,
+	// which help to collect copTasks' runtime stats.
+	copPlanIDs []string
```

Comment:
We can remove this and change the definition of `copStats map[string]map[string][]*CopRuntimeStats` to `copStats []map[string][]*CopRuntimeStats` since we can use the ordinal to find the corresponding element in copStats.

Reply:
Yeah, if this is removed from `selectResult`, we can remove `SelectWithRuntimeStats()` as well.